### DISCUSSION TITLE
Should `.nuxt/tsconfig.server.json` contain `"noEmit": true`?

### DISCUSSION THREAD

**Discussion by rijenkii** [nuxt/nuxt#26458] [Questions]
Currently, `.nuxt/tsconfig.server.json` does not contain `"noEmit": true`, and because of that TypeScript in VSCode is swearing at `.js` config files marked with `@ts-check` inside the project root:
```
Cannot write file '.../tailwind.config.js' because it would overwrite input file.
```

After adding it manually, the error disappeared. Shoud I make an issue?

**Comment by rijenkii** [comment]
For now the following `nuxt.config.ts` does the trick:
```typescript
export default defineNuxtConfig({
  nitro: {
    typescript: {
      tsConfig: {
        compilerOptions: {
          noEmit: true,
        },
      },
    },
  },
});
```

**Reply by xjccc** [reply]
Is it top `typescript.tsConfig`?

[tsConfig](https://nuxt.com/docs/api/nuxt-config#tsconfig)

**Reply by rijenkii** [reply]
Nope, the nitros tsConfig. Main nuxt tsconfig.json contains `noEmit` already, nitros one does not.

**Reply by xjccc** [reply]
Okay.😁
